cmd/genji/shell/command.go

```diff
@@ -60,6 +60,11 @@ func displayTableIndex(db *genji.DB, tableName string) error {
 	}
 	defer res.Close()
 
+	_, err = res.Tx.GetTable(tableName)
```

Comment:
I think we can select from `__genji_tables`.

https://github.com/genjidb/genji/blob/6aa01a11de96f2da7336b25d38c42d6509e9db69/cmd/genji/shell/shell.go#L435

Reply:
You're it's a better way.

Reply:
You're right it's a better way.